api/resolvers/wallet.js

```diff
@@ -206,7 +206,10 @@ export default {
 
       history = history.map(f => {
         if (f.bolt11) {
-          const inv = lnpr.decode(f.bolt11)
+          let inv
+          try {
+            inv = lnpr.decode(f.bolt11)
+          } catch { }
```

Comment:
you can just filter by `FORGOTTEN_HASH` instead of `try/catch` no? This could lead to confusion in the future why we're just ignoring errors.

Reply:
Good point